dex/src/main/scala/com/wavesplatform/dex/model/OrderInfo.scala

```diff
@@ -192,7 +210,21 @@ object OrderInfo {
                            orderType: AcceptedOrderType,
                            avgWeighedPrice: Long,
                            orderVersion: Byte): OrderInfo[S] =
-    Impl(5, side, amount, price, matcherFee, matcherFeeAssetId, timestamp, status, assetPair, orderType, avgWeighedPrice, orderVersion)
+    Impl(
+      5,
+      side,
+      amount,
+      price,
+      matcherFee,
+      matcherFeeAssetId,
+      timestamp,
+      status,
+      assetPair,
+      orderType,
+      avgWeighedPrice,
+      orderVersion,
+      getTotalExecutedAmountOfPriceAsset(status.filledAmount, avgWeighedPrice)
```

Comment:
This is okay for old versions of OrderInfo. But in this task we need the exact information. Probably, the best way to achieve this is to nominator instead of avgWeighedPrice

Reply:
OrderInfo v6 uses `avgWeighedPriceNominator / Order.PriceConstant` (`FillingInfo.totalExecutedAmountOfPriceAsset` field)

Reply:
Interesting... Could we lost some coins because 
```
truncate((avgWeighedPriceNominator + x) / Order.PriceConstant)
!=
truncate(avgWeighedPriceNominator / Order.PriceConstant) + truncate(x / Order.PriceConstant)
```
?

Reply:
Yes we could. So I suggets to divide in the end, that is to leave everything as it is